sycl/test-e2e/Matrix/slm_utils.hpp

```diff
@@ -0,0 +1,81 @@
+//==------------------ slm_utils.hpp  - DPC++ joint_matrix------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+template <unsigned int colsA, unsigned int colsB, unsigned int MCache2,
+          unsigned int NCache2, unsigned int KCache2, unsigned int vnniFactor,
+          size_t SGs, typename TOperand, access::address_space Space>
+inline void
+slm_read_write(multi_ptr<TOperand, Space, access::decorated::yes> pA,
+               multi_ptr<TOperand, Space, access::decorated::yes> pB,
+               local_accessor<TOperand, 2> tileA,
+               local_accessor<TOperand, 2> tileB, sub_group sg, unsigned int k2,
+               size_t m2, size_t n2, size_t sgSize) {
+  size_t slmReadA = KCache2 / sgSize;
```

Comment:
`slmReadA` variable name is not clear. To understand it's meaning, need to go through the logic and find how it is used.
Would be good to indicate it is number of elements per 1 read call.

Reply:
This makes it too long. I will add a comment.

Reply:
`elemsPerRead` should be fine, but I'm also ok with a comment.